test/integration-test.py

```diff
@@ -302,6 +316,44 @@ def main():
             print("\nIssused certificate for domain with bad CAA records")
             die(ExitStatus.NodeFailure)
 
+        if run_node_test("challenge-not-started.com", challenge_types[0], None, "startChallenge")\
+                != ABORT_SUCCESS:
+            print("\nDidn't cleanly abort before attempting challenge")
+            die(ExitStatus.NodeFailure)
+
+        now = datetime.datetime.now()
+        after_grace_period = now + datetime.timedelta(days=+14, minutes=+3)
+
+        target_time = now
+        expected_output = 'Deleted a total of 0 expired pending authorizations'
+        try:
+            out = subprocess.check_output(
+                '''FAKECLOCK=`date -d "%s"` ./bin/expired-authz-purger -db-connect-file test/secrets/purger_dburl -grace-period 168h -yes'''
```

Comment:
This is a good idea, I think it would touch enough unrelated code that it should be probably done in another changeset though.